### PR TITLE
Weekly cleaning up  oryxlinux pool

### PR DESCRIPTION
**Pull request by arroyc**

### CODE REVIEW

File: vsts/pipelines/weeklyCleaning.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
+variables:
```
Comment (kichalla) [inline vsts/pipelines/weeklyCleaning.yml]: nit: Rename the file to like `agentCleanup.yml`? We might want to change the trigger timeline to biweekly in which case this file name is incorrect.

File: vsts/pipelines/weeklyCleaning.yml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        agentName: DDMBLDL020
+    - template: templates/_agentCleanUpJobTemplate.yml
+      parameters:
+        agentName: DDMBLDL021
+      
+    - template: templates/_agentCleanUpJobTemplate.yml
+      parameters:
+        agentName: DDMBLDL022
+
+trigger: none
```
Comment (kichalla) [inline vsts/pipelines/weeklyCleaning.yml]: Seems like you can add scheduled triggers in yaml itself: https://docs.microsoft.com/en-us/azure/devops/pipelines/build/triggers?view=azure-devops&tabs=yaml#scheduled-triggers

File: vsts/pipelines/templates/_agentCleanUpJobTemplate.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+parameters:
+  agentName: ''
+
+jobs:
+- job: Cleaning 
+  displayName: Agent ${{ parameters.agentName }}
+  pool:
+    name: OryxLinux
+    demands:
+    - agent.name -equals ${{ parameters.agentName }}
```
Comment (kichalla) [inline vsts/pipelines/templates/_agentCleanUpJobTemplate.yml]: 👍

File: vsts/pipelines/weeklyCleaning.yml (hunk 2 above)
Comment (arroyc) [reply vsts/pipelines/weeklyCleaning.yml]: as discussed I would do the scheduling using UI

**Pull request merged**